temporal_tasks.py

```
@@ -13,24 +13,75 @@
+    # Extract summary nodes and raw nodes from metadata
+    summary_nodes: list[NodeWithScore] = []
+    raw_nodes: list[NodeWithScore] = []
+
+    for i in range(len(references)):
+        # references is a list of SubQuestionAnswerPair
+        raw_nodes.extend([node for node in references[i].sources])
+
+    # Extract summary nodes from metadata if available
+    # metadata structure: {'Telegram': {'summary_nodes': [node_list]}, 'Discord': {'summary_nodes': [node_list]}, ...}
+    if metadata:
+        for platform_name, platform_metadata in metadata.items():
```

Ruff (0.11.9)

Loop control variable `platform_name` not used within loop body

Rename unused `platform_name` to `_platform_name`

(B007)